index.html

```diff
@@ -10,7 +10,9 @@
 <body>
 
 <div class="container-fluid p-5 bg-primary text-white text-center">
-  <h1>My First Bootstrap Page</h1>
+  <h1>Adding a Landing page message by Carti</h1>
+  
+  <h2>My First Bootstrap Page</h2>
```

Comment:
Could you remove the word "Bootstrap" from

Reply:
Sure I will remove it